### PR TITLE
Update all Utrecht municipality datasets to 2018

### PR DESCRIPTION
**Pull request by redekok**
And keep the older 2016 dataset for Amersfoort in order to maintain the (published) original scenario results

### CODE REVIEW

File: datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad [hunk 1]
```diff
@@ -126,37 +104,59 @@
...
-- number_of_buildings = 9623.0
-- number_of_cars = 98852.0
-- number_of_residences = 65289.0
-- number_of_inhabitants = 153602.0
-- offshore_suitable_for_wind = 448.608030215936
+- number_of_buildings = 4932.0
+- number_of_cars = 102990.0
+- number_of_residences = 66530.0
+- number_of_inhabitants = 155226.0
+- offshore_suitable_for_wind = 12500.0
```
Comment (mabijkerk) [inline datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad:146]: The offshore area suitable for wind seems to have increased strongly. Is this as expected?

Comment (mabijkerk) [reply datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad:146]: It is now the same value for all municipalities: 12500 square kms.

Comment (redekok) [reply datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad:146]: > It is now the same value for all municipalities: 12500 square kms.

Interesting.. This seems to be the case for all 2018 datasets in the dataset manager. @michieldenhaan would you have any idea what's going on here?

Comment (michieldenhaan) [reply datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad:146]: If I remember correctly we first scaled the potential based on number of residences but after receiving complaints that this was too strict in some scenarios we decided to allocate the full NL potential to each municipality (to ensure the slider max is always sufficient)

Comment (redekok) [reply datasets/GM0307_amersfoort/GM0307_amersfoort.derived.ad:146]: That explains, thanks!

**Review by mabijkerk — APPROVED** [review]

**Pull request merged**